doc/Developer Guide/Package Publishing/Readme.md

```diff
@@ -0,0 +1,25 @@
+Package Publishing
+==================
+
+## Plugin Publish Checklist
+This is a list of point to check prior to published your package on [packages.opentap.io](https://packages.opentap.io).
+
+### Update the Package Definition (`package.xml`) file:
+* Provide [`<Description>`](https://doc.opentap.io/Developer%20Guide/Plugin%20Packaging%20and%20Versioning/#description-element) of the package, the following formatting is supported:
+  * `<Status>`: Use one of the following: `Concept` = work has not yet started; `Initial Development` = before 1st release; `Active Development` = 1st release has happened, work is being done on future release; `Maintenance` = no future release planned, but some defects will be fixed.
+  * `<Contacts>`: such as (Developers/Planner/Manager) [Originator, Primary Owner, Secondary Owner] including email.
+  * `<Prerequisites>`: if any. Specify what is required to be able to use this package.
+  * `<Hardware>`: If relevant specify what is supported in terms of hardware/equipment, use product numbers and/or names.
+  * `<Links>`: to direct users to relevant material, e.g. documentation.  
```

Comment:
Perhaps we should leave these formatting options in the linked doc ("Description Element"), instead of listing them here in this checklist. There also seem to be some discrepencies between the options here and in the linked doc.